### ISSUE TITLE
Run 2 doesn't Run, strange bugging.

### ISSUE THREAD

**Issue by zeronobody0** [ruffle-rs/ruffle#1393]
When attempting to play the game Run 2, everything bugs out, sprites are displaying and hiding themselves seemingly at random.

**Comment by Lord-McSweeney** [comment]
The Kongregate version of Run 2 is now playable.

**Comment by waspennator** [comment]
Tried out Run 2 on NG and yanked a copy to run through the ruffle web demo and it seems to work alright when running it for the first time, spitting out these messages in the logs when reaching the title screen but being otherwise playable.

Serialization is not implemented for class other than Object: ByteArrayObject(ByteArrayObject { ptr: 0x13eab1c })

As soon as I tried reloading the page to try the game again, it softlocked right on the title screen breaking with this error.

Error occured when running AVM2 frame script: TypeError: Error # 1009: Cannot access a property or method of a null object reference. (accessing field: length)
